### ISSUE TITLE
Variant builds fail

### ISSUE THREAD

**Issue by minsko** [restorer/gloomy-dungeons-2#13]
The full build cannot be compiled.  Errors occur with `gradlew build` when compiling the`forfdroidWithgplayWoutzeemoteWithkoDebug` variant:

```
 gloomy-dungeons-2/src/main/java/zame/game/engine/Config.java:33: error: cannot find symbol
        public ConfigZeemoteHelper zeemoteHelper = new ConfigZeemoteHelper();
```

**Comment by minsko** [comment]
Fix in Pull Request #14

**Comment by restorer** [comment]
PR merged
